### PR TITLE
Fix the typescript compile issues

### PR DESCRIPTION
**Pull request by elonmitchell**
Problem
=======
Fix the typescript compile issues
[link to JIRA RGR-392](https://usxtech.atlassian.net/browse/RGR-392)

Solution
========
Fixed the typescript compile issues

Screenshots (optional):
-----------------------
Screenshot of Android device

### CODE REVIEW

File: src/hooks/useFreshchat.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export const resetVariantChat = async (): Promise<void> => {
 };
 
 export const useFreshchatInit = (
-  driverId: string,
+  driverId: string | null,
```
Comment (ajp8164) [inline src/hooks/useFreshchat.ts:92]: I believe this was changed in Roger to just be 'string'. I worked on it with Mike - check with him. Might need to make the same change in OC (so this change might not be necessary).

File: src/lib/VariantChat.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ let getNewMessages = (): void => {
 };
 
 const useVariantChat = (
-  driverId: string,
+  driverId: string | null,
```
Comment (ajp8164) [inline src/lib/VariantChat.ts:28]: Same.

**Review by ajp8164 — COMMENTED** [review]
Few comments

File: src/hooks/useFreshchat.ts (hunk 1 above)
Comment (elonmitchell) [reply src/hooks/useFreshchat.ts:92]: @ajp8164 I can see that you changed the `driverId: string | null` to the `driverId || ''` in the Roger app

**Review by ajp8164 — APPROVED** [review]
ACK

**Pull request merged**